src/coreclr/nativeaot/System.Private.CoreLib/src/System/Exception.NativeAot.cs

```diff
@@ -30,10 +30,6 @@ public partial class Exception
 
         private static string? SerializationWatsonBuckets => null;
 
-        // WARNING: We allow diagnostic tools to directly inspect these three members (_message, _innerException and _HResult)
```

Comment:
Delete these comments from everywhere. They were specific to .NET Native for UWP debugger that we do not care about anymore.